modules/twinklefluff.js

```diff
@@ -64,6 +64,8 @@ Twinkle.fluff = function twinklefluff() {
 
 Twinkle.fluff.skipTalk = null;
 Twinkle.fluff.rollbackInPlace = null;
+// String to insert when a username is hidden
+Twinkle.fluff.hiddenName = 'an unknown user';
```

Comment:
Weren't we using `<username hidden>` at some point?

Reply:
In arv, yeah, in #875.  I thought this would be clearer in a sentence-like edit summary